### PR TITLE
[+] Add Hangmster

### LINKED ISSUE #85: Add Hangmster
https://twitter.com/Hangmster 
Need further investigation.

### PR DESCRIPTION
**Pull request by Twinsherry**
- resolve #85

### CODE REVIEW

File: people/Hangmster/page.en.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Hangmster
+info:
+    alias: 仓鼠
+    location: Shijiazhuang, Hebei
+---
+
+Hangmster is an ally of us.
```
Comment (Aquaticat) [inline people/Hangmster/page.en.md:8]: Hangmster is an ally.

没必要写 of us

File: people/Hangmster/page.en.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+They often interacted with friends at Twitter/X, and they were a barrel of laughs. They would also give warm hugs when we needed, console us when we were in anxiety, and mourn for our departed friends.
```
Comment (Aquaticat) [inline people/Hangmster/page.en.md:10]: They often interacted with friends on Twitter/X, bringing us laughter.
They gave warm hugs when we needed, consoled us when we were anxious, and mourned for our departed friends.

File: people/Hangmster/page.en.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Although they seldom mentioned what they loved. However, according to their tweets, they would be attracted by cute things such as little cats. They are fond of Hatsune Miku as well. In the Hatsune Miku 15th anniversary, they expressed their happiness by drawing an illustration:
```
Comment (Aquaticat) [inline people/Hangmster/page.en.md:12]: They seldomly mentioned what they loved.
But according to their tweets, they were interested in cute things such as cats.
They were fond of Hatsune Miku as well.
On the Hatsune Miku 15th anniversary, they expressed their happiness by drawing an illustration:

File: people/Hangmster/page.en.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+<PhotoScroll photos={[ '${path}/photos/fufu.jpg',]} />  
+
+Hangmster always be troubled by sleeplessness and anxiety. They once said that they actually didn't like their body in reality, and diversified self-personality would make them more comfortable.
```
Comment (Aquaticat) [inline people/Hangmster/page.en.md:16]: Hangmster was often troubled by insomnia and anxiety.
They once said that they actually didn't like their body, and their less traditional self-expression online made them more comfortable.

File: people/Hangmster/page.en.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+Unfortunately, although they always encouraged us to live bravely, but they still chose to leave at the beginng of the year.
```
Comment (Aquaticat) [inline people/Hangmster/page.en.md:18]: Unfortunately, although they always encouraged us to live bravely, they still chose to leave at the beginning of the year.

英文不能用 although 和 but 在一个句子里。

File: people/Hangmster/page.en.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Rest well, Hangmster. May you can be yourself as you like.
```
Comment (Aquaticat) [inline people/Hangmster/page.en.md:20]: Rest well, Hangmster. May you can be yourself.

没必要写 as you like

File: people/Hangmster/page.en.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+
```
Comment (Aquaticat) [inline people/Hangmster/page.en.md:22]: 在文件结尾加一个空行。

**Review by Aquaticat — CHANGES_REQUESTED** [review]

File: people/Hangmster/page.en.md (hunk 1 above)
Comment (Twinsherry) [reply people/Hangmster/page.en.md:8]: 嘛，一直这么写其实是呼应网站标题来着，强调“我们之中的某位”，不过要修改的话肯定就是 our ally 了，an ally比较奇怪。

File: people/Hangmster/page.en.md (hunk 4 above)
Comment (Twinsherry) [reply people/Hangmster/page.en.md:16]: Anxiety后面需要加disorder嘛，我第一稿的失眠症用的是insomnia的专有词汇，但是后来想想并列情况下是不是统一口语用词比较好。

File: people/Hangmster/page.en.md (hunk 1 above)
Comment (Aquaticat) [reply people/Hangmster/page.en.md]: 嗯，our ally 更好。不过不用强调这个。

File: people/Hangmster/page.en.md (hunk 4 above)
Comment (Aquaticat) [reply people/Hangmster/page.en.md]: 不用加 disorder。insomnia 也不算 official，不用在意并列都用口语表达。

**Comment by Twinsherry** [comment]
已经修改了目前的英文版本，由于目前没有新增内容来源，已加挂标签。

**Review by Aquaticat — APPROVED** [review]

**Review by LS-KR — APPROVED** [review]

**Pull request merged**